src/Features/Core/Portable/ExtractMethod/MethodExtractor.AnalyzerResult.cs

```diff
@@ -162,6 +138,17 @@ public IEnumerable<VariableInfo> GetVariablesToSplitOrMoveOutToCallSite(Cancella
                 return _variables.Where(v => v.GetDeclarationBehavior(cancellationToken) is DeclarationBehavior.SplitOut or
                                                  DeclarationBehavior.MoveOut);
             }
+
+            public VariableInfo GetOutermostVariableToMoveIntoMethodDefinition(CancellationToken cancellationToken)
+            {
+                using var _ = ArrayBuilder<VariableInfo>.GetInstance(out var variables);
+                variables.AddRange(this.GetVariablesToMoveIntoMethodDefinition(cancellationToken));
```

Comment:
>this.GetVariablesToMoveIntoMethodDefinition(cancellationToken)) [](http://example.com/codeflow?start=35&length=63)

nit: no need to sort, could just loop over and find the min, right?

Reply:
Yup. Will fix!

Reply:
done in mop-up pr.